### PR TITLE
Fix for out of memory execption in js tests

### PR DESCRIPTION
**Pull request by Pask423**

### CODE REVIEW

**Comment by adamw** [comment]
🥳 would be great if this would actually solve the memory problems

File: build.sbt [hunk 1]
```diff
@@ -43,6 +43,9 @@ val commonJvmSettings = commonSettings ++ Seq(
 )
 
 val commonJsSettings = commonSettings ++ Seq(
+  scalaJSLinkerConfig ~= {
+    _.withBatchMode(true).withParallel(false)
```
Comment (adamw) [inline build.sbt:47]: maybe add (directly to master) a comment explaining why we need this and how does this fix the issue?

**Comment by Pask423** [comment]
We will see as for now it works as expected I tired to run CI several times and everything seemed to be ok.

File: build.sbt (hunk 1 above)
Comment (Pask423) [reply build.sbt:47]: I will add it

**Pull request merged**